extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.js

```diff
@@ -609,10 +609,11 @@ angular.module('PaperUI.controllers.configuration', [ 'PaperUI.constants' ]).con
             dict.bridgeUID = thing.bridgeUID
             update = true;
         }
-        if (update) {
+        originalThing.configuration = thing.configuration = {};
+        if (JSON.stringify(originalThing) !== JSON.stringify(thing)) {
             thingService.update({
                 thingUID : thing.UID
-            }, dict);
+            }, thing);
```

Comment:
Does "thingService.update" ignore the configuration of a thing?
You are using thing as argument and set thing.configuration to {} above.
As you are more familiar with that JS part, I assume it is correct and no configuration get's lost. The empty configuration will be identified as "don't change configuration" and be ignored. Correct?